### PR TITLE
IDCs: fix warning message

### PR DESCRIPTION
**Pull request by matthias-kleiner**

### CODE REVIEW

**Comment by matthias-kleiner** [comment]
can this be merged, please? When the warning was triggered before this PR it caused a crash of the workflow due to the missing parameter

**Comment by chiarazampolli** [comment]
@shahor02 , @martenole , could you merge?

File: Detectors/TPC/workflow/include/TPCWorkflow/TPCFactorizeIDCSpec.h [hunk 1]
```diff
@@ -81,19 +81,21 @@ class TPCFactorizeIDCSpec : public o2::framework::Task
...
+      LOGP(warning, "firstTF not Found!!! Found valid inputs {}. Setting {} as first TF", pc.inputs().countValidInputs(), mTFFirst);
     }
 
+    const unsigned int relTF = (mTFFirst == -1) ? 0 : currTF - mTFFirst;
+
     // loop over input data
     for (auto& ref : InputRecordWalker(pc.inputs(), mFilter)) {
       ++mProcessedCRUs;
-      if (relTF >= mIDCFactorization.getNTimeframes()) {
+      if ((relTF >= mIDCFactorization.getNTimeframes()) || (relTF < 0)) {
```
Comment (martenole) [inline Detectors/TPC/workflow/include/TPCWorkflow/TPCFactorizeIDCSpec.h:98]: `relTF` is unsigned and will never be smaller than 0. Is it also OK without the check or do you want to check above whether `currTF >= mTFFirst`?

Comment (matthias-kleiner) [reply Detectors/TPC/workflow/include/TPCWorkflow/TPCFactorizeIDCSpec.h:98]: Thanks I missed that, I will do a check than for `currTF >= mTFFirst`

Comment (matthias-kleiner) [reply Detectors/TPC/workflow/include/TPCWorkflow/TPCFactorizeIDCSpec.h:98]: I just changed the datatype from `unsigned int` to `long`, which would be the proper solution, also for the other places where `relTF` is used

**Review by martenole — APPROVED** [review]

**Comment by matthias-kleiner** [comment]
Can this be merged now? The error in the fullCI seems unrelated

**Pull request merged**